hybrid/core.py

```diff
@@ -382,7 +388,7 @@ def run(self, state, defer=True):
 
     def stop(self):
         """Terminate an iteration of an instantiated :class:`Runnable`."""
```

Comment:
"""Terminate an iteration of an instantiated :class:`Runnable`.

Calls `halt`. Override `halt` to stop a blocking call of `next`.  
"""